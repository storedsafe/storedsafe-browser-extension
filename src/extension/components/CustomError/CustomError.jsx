import React from 'react';
import PropTypes from 'prop-types';

function CustomError({ message }) {
  return (
    <p className="error">{message}</p>
  );
}

CustomError.propTypes = {
  message: PropTypes.string.isRequired,
};

export default CustomError;
