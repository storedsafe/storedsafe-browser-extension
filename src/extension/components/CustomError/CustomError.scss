@use '../../base.scss';

.error {
  color: base.$error;
}
