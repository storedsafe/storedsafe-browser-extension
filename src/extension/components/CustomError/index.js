import CustomError from './CustomError';
import './CustomError.scss';

export default CustomError;
