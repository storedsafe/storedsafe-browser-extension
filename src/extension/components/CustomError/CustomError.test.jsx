import React from 'react';
import { shallow } from 'enzyme';
import CustomError from '.';

test('<CustomError />', () => {
  const message = 'my error';
  const wrapper = shallow(<CustomError message={message} />);
  expect(wrapper.props().children).toContain(message);
  expect(wrapper).toMatchSnapshot();
});
