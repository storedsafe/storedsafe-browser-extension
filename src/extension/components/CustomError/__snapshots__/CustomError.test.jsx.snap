// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomError /> 1`] = `ShallowWrapper {}`;
